### DISCUSSION TITLE
MQTT doesnt refres stat/POWER correctly for RBG LED controller

### DISCUSSION THREAD

**Discussion by BitsoftHR** [arendst/Tasmota#18152] [Ideas]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._
I have Athom controller for RGB LED strip with prefleshed Tasmota. I have updated it to latest Tasmota version. There is problem with MQTT state stat/POWER. When controller is OFF and you send command cmnd/Color = #xxxxxx (any color) than controller automatically switches on and stat/RESULT changles to POWER: ON in json but stat/POWER is OFF and didnt changet to ON

Also i think it would be better that Color in RESULT json should have # prefix so it would be more compatible with other softwares but that isnt a bug

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [ ] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [ ] Device used (e.g., Sonoff Basic): Athom Light Strip Controller RGB, Tasmota
- [ ] Tasmota binary firmware version number used: 12.4.0
  - [x] Pre-compiled
  - [ ] Self-compiled
- [ ] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_
send  cmnd/POWER = OFF
send  cmnd/Color #xxxxxx  (any color)

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._
when controller is on, stat/POWER should be ON

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._
![image](https://user-images.githubusercontent.com/115004546/224260783-5bc00dd6-1ba6-4216-9259-2f2d8c0c3353.png)

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by Jason2866** [comment]
Moving to discussions, since it is a feature request. The current behaviour is as designed.
